RedHatInsights/compliance-frontend
# feat(TableTools): RHICOMPL-3769 - Allow switching between tree and list view

This adds the ability for users to switch between tree and list view of a rules table.


**How to test:**

(see also https://github.com/RedHatInsights/compliance-frontend/pull/1945 to enable the features)

1) Open any page that contains a Rules Table (for example the policy details rules tab)
2) Try switching between list and tree view.
3) Switch to the tree view and filter. This will switch automatically to the list view
4) Expand an item and switch between list and tree view verifying the expansion is maintained.
5) Select an item and switch views to verify the selection is maintained.
6) Try to break it. 

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
